pkgs/development/python-modules/bcrypt/3_1.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, isPyPy, fetchPypi
+, cffi, pycparser, mock, pytest, py, six }:
+
+with stdenv.lib;
+
+buildPythonPackage rec {
+  version = "3.1.7";
+  pname = "bcrypt";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0b0069c752ec14172c5f78208f1863d7ad6755a6fae6fe76ec2c80d13be41e42";
+  };
+  buildInputs = [ pycparser mock pytest py ];
+  propagatedBuildInputs = [ six ] ++ optional (!isPyPy) cffi;
+
+  meta = {
+    maintainers = with maintainers; [ domenkozar ];
+    description = "Modern password hashing for your software and your servers";
+    license = licenses.asl20;
+    homepage = "https://github.com/pyca/bcrypt/";
```

These could be reordered.
```suggestion
    description = "Modern password hashing for your software and your servers";
    homepage = "https://github.com/pyca/bcrypt/";
    license = licenses.asl20;
    maintainers = with maintainers; [ domenkozar ];
```